core/app/models/spree/variant.rb

```diff
@@ -38,7 +38,9 @@ class Variant < Spree::Base
 
     validate :check_price
 
-    validates :option_values, presence: true, unless: :is_master?
+    ## Override +option_values_required?+ if it is not necessary
+    ## to have option_values present to create a variant.
+    validates :option_values, presence: true, if: :option_values_required?, unless: :is_master?
```

Comment:
@jitendra - I think we shouldn't move is_master? to option_values_required? because is_master? is like a necessary condition and option_values_required? is there to provide a behaviour to bypass the validation.

We can discuss it in more detail in person.